src/alire/os_macos/alire-platforms-current__macos.adb

```diff
@@ -9,7 +9,7 @@ package body Alire.Platforms.Current is
    ------------------
 
    function Detected_Distribution return Platforms.Distributions is
-      (Platforms.Distro_Unknown);
+      (Platforms.MacOS);
```

Comment:
This is where you detect the `HOMEBREW_PREFIX` env variable and return `Platforms.Homebew` if present, `Platforms.Distro_Unknown` otherwise.